src/components/sidebar/component.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import { Display, JustifyContent, Layout } from "ui";
+import "./style.scss";
+
+type PublicProps = {};
+
+const SideBar: React.FunctionComponent<PublicProps> = () => {
+  return (
+    <Layout
+      display={Display.Flex}
```

Comment:
another basic CSS question: when should we be using flexbox vs grid? I see everything as flexbox, but i feel like in the past I've heard statements like "grid is the way bro"

Reply:
this is mostly just a personal choice because I haven't worked with the new grid properties 😂. I think we probably should refactor this to use a grid for layouts specifically.